jsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=759670
+	// for the documentation about the jsconfig.json format
```

Comment:
Also, what is this file for, is it from your editor, if so, can you add the filename into the `.gitignore`